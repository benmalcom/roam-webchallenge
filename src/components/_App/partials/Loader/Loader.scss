.loader {
  margin-top: 50px;
  p {
    text-align: center;
  }
}
