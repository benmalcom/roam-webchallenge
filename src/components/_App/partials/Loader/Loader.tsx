import React from 'react';
import './Loader.scss';
export default () => (
	<div className="loader col-md-12 text-center text-muted">
		<p><i className="fa fa-spinner fa-spin fa-3x"/></p>
		<p>Please wait...</p>
	</div>);
