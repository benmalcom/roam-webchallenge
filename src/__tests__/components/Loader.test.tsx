import React from 'react';
import Loader from '../../components/_App/partials/Loader/Loader';
import { shallow } from 'enzyme';
describe('<Loader />', () => {
	test('Component renders properly', () => {
		const component = shallow(<Loader />);
		expect(component).toMatchSnapshot();
	});
});
