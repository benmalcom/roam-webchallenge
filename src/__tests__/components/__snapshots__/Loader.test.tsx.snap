// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Component renders properly 1`] = `
<div
  className="loader col-md-12 text-center text-muted"
>
  <p>
    <i
      className="fa fa-spinner fa-spin fa-3x"
    />
  </p>
  <p>
    Please wait...
  </p>
</div>
`;
